pkg/pillar/hypervisor/kvm.go

```diff
@@ -169,6 +169,7 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   logfile = "/dev/fd/1"
   logappend = "on"
 
+{{- if .DomainConfig.EnableVncShimVm}}
```

Comment:
This implies that it needs to be enabled before the app instance is booted. That needs to be documented.

Reply:
Documentation is a very good point. You also hide the prime console chardev under this "if" branch, which means it won't be created on the guest side if vnc for shim is disabled. Probably you meant to hide the "[chardev "charserial2"]", line 177.

Reply:
nice catch @rouming , thank you

Reply:
Done